DigitalLearningSolutions.Data.Tests/Services/SectionContentServiceTests.cs

```diff
@@ -0,0 +1,110 @@
+            sectionContentService = new SectionContentService(connection, logger);
+        }
+
+        [Test]
+        public void Get_section_content_should_return_section_content()
+        {
+            // When
+            const int customisationId = 15853;
+            const int progressId = 173218;
+            const int sectionId = 382;
+            var result = sectionContentService.GetSectionContent(customisationId, progressId, sectionId);
+
+            // Then
+            var expectedSectionContent = new SectionContent(
+                "Erin Test 01",
+                "Office 2013 Essentials for the Workplace",
+                "Working with Microsoft Office applications",
+                0,
+                28,
+                true,
+                0
+            );
+            result.Should().BeEquivalentTo(expectedSectionContent);
+        }
+
+        [Test]
+        public void Get_section_content_should_return_null_if_customisation_id_is_invalid()
+        {
+            //When
+            const int customisationId = 0;
+            const int progressId = 173218;
+            const int sectionId = 382;
+            var result = sectionContentService.GetSectionContent(customisationId, progressId, sectionId);
+
+            // Then
+            result.Should().BeNull();
+        }
+
+        [Test]
+        public void Get_section_content_should_return_null_if_progress_id_is_invalid()
+        {
+            //When
+            const int customisationId = 15853;
+            const int progressId = 0;
+            const int sectionId = 382;
+            var result = sectionContentService.GetSectionContent(customisationId, progressId, sectionId);
+
+            // Then
+            result.Should().BeNull();
+        }
+
+        [Test]
+        public void Get_section_content_should_return_null_if_section_id_is_invalid()
+        {
+            //When
+            const int customisationId = 15853;
+            const int progressId = 173218;
+            const int sectionId = 0;
+            var result = sectionContentService.GetSectionContent(customisationId, progressId, sectionId);
+
+            // Then
+            result.Should().BeNull();
+        }
+
+        [Test]
+        public void Get_section_content_should_return_correct_section_when_percent_complete_is_not_zero()
+        {
+            // When
+            const int customisationId = 19262;
+            const int progressId = 201058;
+            const int sectionId = 1011;
+            var result = sectionContentService.GetSectionContent(customisationId, progressId, sectionId);
+
+            // Then
+            var expectedSectionContent = new SectionContent(
+                "Testing",
+                "Excel 2013 for the Workplace",
+                "Entering data",
+                4,
+                28,
+                true,
+                25
+            );
+            result.Should().BeEquivalentTo(expectedSectionContent);
+        }
+    }
```

Comment:
We should also have a test for if there's no progress record, or the progress record is invalid (due to having removed date or system reset) 🙂